code/modules/mob_spawn/ghost_roles/mining_roles.dm

```diff
@@ -323,3 +323,36 @@
 /obj/item/clothing/mask/chameleon/gps/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/gps, "Encrypted Signal")
+
+/obj/effect/mob_spawn/ghost_role/human/plasmaman
+	name = "plasma filled pod"
+	desc = "A sleeper retrofitted to use plasma instead of oxygen."
+	prompt_name = "plasmaman survivor"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "psykerpod"
+	density = TRUE
+	mob_species = /datum/species/plasma
+	you_are_text = "You are a plasmaman, a sentient colony of fungi created by one of the plasma pits outside."
+	flavour_text = "Dispite not knowing your past or who you once were, you were able to keep information even through your rebirth \
+	The language you remember feels forign on your tongue, yet you still speak it."
+
+/obj/effect/mob_spawn/ghost_role/human/plasmaman/Initialize(mapload)
+	. = ..()
+	mob_name = pick("Hydrogen", "Helium", "Lithium", "Beryllium", "Boron", "Carbon", "Nitrogen", "Oxygen", \
```

Comment:
Rather than have this massive `pick`, just do `mob_name = pick(GLOB.plasmaman_names)`